ci/Jenkinsfile_utils.groovy

```diff
@@ -67,7 +67,7 @@ def pack_lib(name, libs, include_gcov_data = false) {
   sh returnStatus: true, script: """
 set +e
 echo "Packing ${libs} into ${name}"
-echo ${libs} | sed -e 's/,/ /g' | xargs md5sum
+for i in \$(echo ${libs} | sed -e 's/,/ /g'); do md5sum \$i; done
```

Comment:
Isn't this the same?

Reply:
This is to handle glob patterns in `$libs`. The names of the shared library files produced by cython depend on the python version and platform. Using glob pattterns is convenient.